pkg/cluster/install.go

```diff
@@ -156,6 +161,17 @@ func (m *manager) adminUpdate() []steps.Step {
 	return toRun
 }
 
+func (m *manager) shouldUpdateOperator() bool {
+	runningVersion, err := semver.NewVersion(m.doc.OpenShiftCluster.Properties.ClusterProfile.Version)
+	if err != nil {
+		return false
+	}
+
+	cutoffVersion := semver.New(operatorCutoffVersion)
+
+	return cutoffVersion.LessThan(*runningVersion)
```

Comment:
nit? `LessThan` will return false if `runningVersion` is `4.7.0`, right? So we're excluding 4.7.0? is that intentional?

Reply:
Confirmed. I'll get a fix up for this.

Reply:
fixed